observability/opentelemetry-extension/src/main/java/org/wildfly/extension/opentelemetry/deployment/OpenTelemetryExtensionLogger.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * Copyright 2021 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.extension.opentelemetry.deployment;
+
+import static org.jboss.logging.Logger.Level.ERROR;
+import static org.jboss.logging.Logger.Level.INFO;
+
+import org.jboss.as.server.deployment.DeploymentUnitProcessingException;
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "WFLYOTELEXT", length = 4)
+public interface OpenTelemetryExtensionLogger extends BasicLogger {
+    OpenTelemetryExtensionLogger OTEL_LOGGER = Logger.getMessageLogger(OpenTelemetryExtensionLogger.class,
+            OpenTelemetryExtensionLogger.class.getPackage().getName());
+
+    @LogMessage(level = INFO)
+    @Message(id = 1, value = "Activating OpenTelemetry Subsystem")
+    void activatingSubsystem();
+
+    @LogMessage(level = INFO) // DEBUG
```

Comment:
DEBUG :)

Note that if it is below INFO it doesn't need to be in this class, and not having it here saves the need to translate it.